web/wp-content/themes/borealis-2021/inc/custom-post-types/research-blogs-cpt.php

```diff
@@ -0,0 +1,60 @@
+<?php 
+
+if (!function_exists('pg_register_research_blog_cpt')) {
+    /**
+     * Creates a Location Custom Post Type with a services custom taxonomy and custom metadata
+     */
+    function pg_register_research_blog_cpt() {
+        // Arguments are:
+        // - slug.
+        // - single.
+        // - plural.
+        // - args (an optional array to overwrite any default settings).
+        $namespace = pg_get_namespace();
+        $Research_Blogs_CPT = new PG_Custom_Post_Type('research-blogs', 'Research Blog', 'Research Blogs', array('icon' => 'dashicons-category', 'has_archive' => 'false', 'template' => [ [$namespace . '/research-blog-meta-block'], [ $namespace . '/body-copy']]));
```

Comment:
@Braedencraig you can remove the 'template' entry from the args array.

Reply:
Also, I would suggest using just research rather than research blog.

Reply:
Ignore that first comment. I would remove the body copy block from the template. All good to keep the research-blogs meta block